src/components/AssetLogo/AssetLogoWithIcon.tsx

```diff
@@ -1,25 +1,27 @@
-import React, { FC, ReactNode } from 'react';
 import { Options } from '@popperjs/core';
-import { AssetLogo, AssetLogoProps } from './AssetLogo';
-import { BoxProps, Box } from '../Box/Box';
-import { Tooltip } from '../Tooltip/Tooltip';
+import React, { FC, ReactNode } from 'react';
+import { Box, BoxProps } from '../Box/Box';
 import { Flex } from '../Flex/Flex';
-import { Icon } from '../Icon/Icon';
+import { Icon, IIcon } from '../Icon/Icon';
 import { Text } from '../Text/Text';
-import { iconSmartMini } from '../../icons/smartMini';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { AssetLogo, AssetLogoProps } from './AssetLogo';
 
-type SmartAssetLogoProps = BoxProps &
+type AssetLogoWithIconProps = BoxProps &
     AssetLogoProps &
-    Record<'popperOptions', Partial<Options>> &
-    Record<'isSmart', boolean>;
+    Record<'popperOptions', Partial<Options>> & {
+        showIcon: boolean;
```

Comment:
It seems like redundant prop. `showIcon` can't be true w/o `icon`.

Reply:
done